app/common/renderer/utils/env-utils.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Recursively interpolate environment variables in an object
+ * @param {*} obj - The object to interpolate
+ * @param {Array} envVars - Array of environment variables [{key: string, value: string}]
+ * @returns {*} - The interpolated object
+ */
+export function interpolateEnvironmentVariables(obj, envVars) {
```

Comment:
what is the purpose of this method?

Reply:
this method helps in replacing the ${asdf} to actual value before sending to create session.